client/rhel/dnf-plugin-spacewalk/dnf-plugin-spacewalk.changes

```diff
@@ -1,3 +1,7 @@
+- Sync changes from Spacewalk
```

Comment:
Do we need a revival of your famous bug?

Reply:
FMPOV: No, unless we feel our SUSE Manager submissions can be rejected. What's your opinion?

Reply:
If we do not need to submit it, no need for that.